Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using LitJson;
+using System.IO;
+using UnityEngine.SceneManagement;
+
+public class GameManager : MonoBehaviour
+{
+    public static GameManager instance = null;
+
+    public string readJson;
+
+    void Awake()
+    {
+        if (instance == null)
+            instance = this;
+
+        else if (instance != this)
+            Destroy(gameObject);
+
+        DontDestroyOnLoad(gameObject);
```

Comment:
Please explain why you DontDestroyOnLoad gameObject here not this?

Reply:
This is to maintain class and inner data when change scene. When change scene, every existing object is destroyed. But if you use 'DontDestroyOnLoad', that object is not destroyed.